help/destinations/catalog/data-management/zetamarketingplatform.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Zeta Marketing Platform
```

Comment:
A couple of file management issues:

- Please rename the doc page to `zeta-marketing-platform.md`.
- Please rename the `assets/catalog/data-management-platform/zmp-extension/` folder to `assets/catalog/data-management-platform/zeta-marketing-platform/`